unit/Makefile

```diff
@@ -69,7 +69,6 @@ BMC_DEPS =../src/cbmc/all_properties$(OBJEXT) \
           ../src/cbmc/cbmc_solvers$(OBJEXT) \
           ../src/cbmc/counterexample_beautification$(OBJEXT) \
           ../src/cbmc/fault_localization$(OBJEXT) \
-          ../src/cbmc/show_vcc$(OBJEXT) \
```

Comment:
is the moved one not needed here ?

Reply:
The moved one is dragged in implicitly via `goto-symex$(LIBEXT)`.